shared/containerwriter/container_tar_writer.go

```diff
@@ -61,12 +61,12 @@ func (ctw *ContainerTarWriter) WriteFile(offset int, path string, fi os.FileInfo
 		return fmt.Errorf("failed to get file stat: %s", err)
 	}
 
-	// Unshift the id under /rootfs/ for unpriv containers
-	if strings.HasPrefix(hdr.Name, "/rootfs") {
+	// Unshift the id under rootfs/ for unpriv containers
+	if strings.HasPrefix(hdr.Name, "rootfs") {
```

Comment:
@stgraber I think this was just a simple bug that stopped the prefix match from working. Now I can see the tarballs created do have the UID/GIDs unshifted.

Reply:
Oh, interesting, I didn't remember us having shifting logic in the tar writing code at all :)